Shopipy.ProductManagement/Mappings/ProductMappingProfile.cs

```diff
@@ -0,0 +1,26 @@
+﻿using AutoMapper;
+using Shopipy.Persistence.Models;
+using Shopipy.ProductManagement.DTOs;
+
+namespace Shopipy.ProductManagement.Mappings
+{
+    public class ProductMappingProfile : Profile
+    {
+        public ProductMappingProfile()
+        {
+            CreateMap<Product, ProductResponseDTO>();
+
+            CreateMap<ProductRequestDTO, Product>()
+                .ForMember(dest => dest.BusinessId, opt => opt.Ignore()) 
+                .ForMember(dest => dest.ProductId, opt => opt.Ignore()) 
+                .ForMember(dest => dest.CreatedAt, opt => opt.Ignore()) 
+                .ForMember(dest => dest.UpdatedAt, opt => opt.Ignore())
+                .ForMember(dest => dest.CategoryId, opt => opt.MapFrom(src => src.CategoryId))
```

Comment:
As konstantinas previously mentioned, whenever you are using src => src.sameNameAsDest variable, you dont need these lines, because automapper does it for you.